solutions/src/test/java/ru/sberbank/school/task08/JacksonSerializeableManagerTest.java

```diff
@@ -0,0 +1,62 @@
+package ru.sberbank.school.task08;
+
+
+import org.junit.Assert;
+import ru.sberbank.school.task08.state.GameObject;
+import ru.sberbank.school.task08.state.InstantiatableEntity;
+import org.junit.Before;
+import org.junit.Test;
+import ru.sberbank.school.task08.state.MapState;
+import ru.sberbank.school.task08.state.Savable;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class JacksonSerializeableManagerTest {
+    private MapState mapState;
+    private JacksonSerializeableManager manager;
+    private String path;
+    private String fileName;
+
+    public JacksonSerializeableManagerTest() {
+        path = "./src/test/java/ru/sberbank/school/task08";
+        fileName = "game.bin";
+    }
+
+    @Before
+    public void initialize () {
```

Comment:
&#39;(&#39; is preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-71991137>*